x-pack/plugins/apm/public/components/app/Correlations/LatencyCorrelations.tsx

```diff
@@ -94,8 +111,42 @@ export function LatencyCorrelations() {
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiAccordion id="accordion" buttonContent="Customize">
+            <EuiFlexGroup>
+              <EuiFlexItem grow={1}>
+                <EuiFormRow label="Threshold">
+                  <EuiFieldNumber
+                    value={durationPercentile}
+                    onChange={(e) =>
+                      setDurationPercentile(e.currentTarget.value)
+                    }
+                  />
+                </EuiFormRow>
+              </EuiFlexItem>
+              <EuiFlexItem grow={4}>
+                <EuiFormRow
+                  fullWidth={true}
+                  label="Field"
+                  helpText="Fields to analyse for significant terms"
+                >
+                  <EuiComboBox
+                    fullWidth={true}
+                    placeholder="Select or create options"
+                    selectedOptions={fieldNames}
+                    onChange={setFieldNames}
+                    onCreateOption={(term) => {
+                      setFieldNames((names) => [...names, { label: term }]);
+                    }}
+                  />
+                </EuiFormRow>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiAccordion>
+        </EuiFlexItem>
         <EuiFlexItem>
           <SignificantTermsTable
+            cardinalityColumnName="# of slow transactions"
```

Comment:
Are we adding i18n at this stage?

Reply:
I thought about it and opted not to. There's a good chance that all of the copy will be changed anyway (I made all of it up) - so translators will have to do this twice. Not sure what the official guidelines around experimental features are.